src/mixin/pending-container.js

```diff
@@ -0,0 +1,57 @@
+
+export const PendingContainer = superclass => class extends superclass {
```

Comment:
Does it make sense for this to be a mixin?

Reply:
Sorry, the name was throwing me off - this is already a mixin.  :)  I think it should be called `PendingContainerMixin`.

Reply:
I hadn't initially considered using an element. Was basically just copying the PendingContainer  mixin from Justin's demos.  I'd probably like to leave it as a mixin for now whilst we try it out in the new activity editor.  If we find that everyone wants to use a helper element at the top level, then that element could extend this mixin and become the standard way for people to use it.

Reply:
Yep I totally agree with it being a mixin, but I think it should be renamed.